cluster/gce/configure-vm.sh

```diff
@@ -50,6 +50,12 @@ function ensure-install-dir() {
   cd ${INSTALL_DIR}
 }
 
+function salt-set-test-grain() {
```

Comment:
Can we just add a single CONNECTION_TIMEOUT variable to the ENV vars we pack instead of copying everything for a test configuration?